packages/provider/src/app/network.ts

```diff
@@ -143,9 +143,10 @@ export function getUrl(
   // List of publically available urls to use
   // TODO(mark): in this case, turn off calls for `eth_getChainId`
   switch (network ? network.name : 'unknown') {
-    case 'main':
-      host = '' // TODO: once the url of mainnet is known
+    case 'gorli':
```

Comment:
Lol so it looks like the best way to spell this is actually `goerli` because the ö is actually "oe" -- which is btw why we sometimes call the OVM the ÖVM 😄